language/Methods.md

```diff
@@ -105,16 +105,23 @@ DoSomething(7, 8, 9, 10);
 ZScript includes an extra method type for descendents of `Actor` called
 *actions*, which are intended to be run from actor states and give extra
 information to the function. Action functions change the meaning of the `self`
-parameter and pass in `invoker` and `stateinfo` parameters as well. `stateinfo`
-refers to the `State` which this action was called from. Here is a chart for
-the meanings of the `self` and `invoker` variables under each scope:
-
-| Scope     | `self` meaning                                                  | `invoker` meaning |
-| -----     | --------------                                                  | ----------------- |
-| None      | The actor this function operates on, ambiguous in some contexts | State owner       |
-| `actor`   | The actor                                                       | The actor         |
-| `item`    | Item owner                                                      | Item itself       |
-| `overlay` | Weapon owner                                                    | Weapon itself     |
-| `weapon`  | Weapon owner                                                    | Weapon itself     |
+parameter and may pass in `invoker` and `stateinfo` parameters as well. `stateinfo`
+refers to the `State` which this action was called from.
+
+Action scopes are a flag field and as such an action function can have multiple scopes.
+The only scopes that actually set the internal action function flag are `item`, `overlay` and `weapon`,
+so action functions without one of these do not act much like action functions, and they do not get an
+`invoker` or `stateinfo` pointer.
```

Comment:
Preferably this would also be rephrased as something like "Actor scope and un-scoped action functions do not have `invoker` or `stateinfo` parameters and as such are not often useful." Followed by an explanation of what these two do and in what circumstances they may be useful.